code/game/machinery/kitchen/smartfridge.dm

```diff
@@ -304,11 +304,13 @@
 		return dump_bag(O, user)
 
 /obj/machinery/smartfridge/proc/insert_item(var/obj/item/O)
-	var/datum/fridge_pile/thisPile = piles[O.name]
+	var/formatted_name = format_text(O.name)
+	var/datum/fridge_pile/thisPile = piles[formatted_name]
 	if(istype(thisPile))
 		thisPile.addAmount(1)
 	else
-		piles[O.name] = new/datum/fridge_pile(O.name, src, 1, costly_bicon(O))
+		piles[formatted_name] = new/datum/fridge_pile(formatted_name, src, 1, costly_bicon(O))
+	updateUsrDialog()
```

Comment:
Wouldn't this get called once for each item, if you're dumping a bag into the fridge? Probably not great

Reply:
Meh it's so fast it's not even profilable when dumping a chock full pill collector so it really really doesn't matter anyways